kiwi/filesystem/isofs.py

```diff
@@ -61,3 +67,19 @@ def create_on_file(self, filename, label=None, exclude=None):
         iso.relocate_boot_catalog(filename)
         iso.fix_boot_catalog(filename)
         return hybrid_offset
+
+    def _find_iso_creation_tool(self):
+        """
+        There are tools by J.Schilling and tools from the community
+        Depending on what is installed a decision needs to be made
+        """
+        iso_creation_tools = [
+            '/usr/bin/mkisofs', '/usr/bin/genisoimage'
+        ]
+        for tool in iso_creation_tools:
+            if os.path.exists(tool):
+                return tool
+        raise KiwiIsoToolError(
+            'No iso creation tool found, searched for: %s' %
+            iso_creation_tools
+        )
```

Comment:
Umhmh... While it is probably always in `/usr/bin`, I wouldn't do that. Instead, I would look for `PATH` variable and then look if the _executable_ is there. Something (:boom: **Untested!**) like:
```python
def _find_isomake(self):
    binaries = ["mkisofs", "genisoimage"]
    paths = os.environ.get("PATH").split(os.pathsep)
    for executable in binaries:
        for path in paths:
            location = os.path.join(path, executable)
            if os.path.exists(location):
                return location
    raise KiwiIsoToolError("Karamba! :-(")
```